sentry-core/src/profiling.rs

```diff
@@ -241,13 +243,12 @@ fn get_profile_from_report(
             _ => "".to_string(),
         },
 
-        event_id: uuid::Uuid::new_v4(), // double check this
-        release: format!(
-            "{} ({})",
-            env!("CARGO_PKG_VERSION"),
-            build_id::get().to_simple(),
-        ),
-        timestamp: DateTime::from(rep.timing.start_time),
+        event_id: uuid::Uuid::new_v4(),
+        release: transaction
+            .release
+            .as_ref()
+            .map_or("".to_string(), |r| -> String { r.to_string() }),
```

Comment:
maybe `transaction.release.unwrap_or_default().into()` would be easier to read here.

Reply:
If I use that I get this error

```
cannot move out of `transaction.release` which is behind a shared reference
move occurs because `transaction.release` has type `std::option::Option<std::borrow::Cow<'_, str>>`, which does not implement the `Copy` trait
```

So I'd have to use a `clone`. Not sure what would be better frankly.

Reply:
ah yes, `clone()` absolutely. you cant an allocated `String` anyway.